client/src/client_sync/v17/wallet.rs

```diff
@@ -756,7 +765,11 @@ macro_rules! impl_client_v17__wallet_process_psbt {
     () => {
         impl Client {
             pub fn wallet_process_psbt(&self, psbt: &bitcoin::Psbt) -> Result<WalletProcessPsbt> {
-                self.call("walletprocesspsbt", &[into_json(psbt)?])
+                // Core expects the PSBT as a base64 string argument (same representation
+                // used by `finalizepsbt`). Serializing the struct with `into_json` produced
+                // an object which Core rejected ("Expected type string, got object").
```

Comment:
This is a known issue, I ran into it myself already. PSBT `serde` serialization does not use base64. I thought we had a tracking issue in `rust-bitcoin` but I couldn't find it just now.

This one kind of covers it: https://github.com/rust-bitcoin/rust-bitcoin/issues/765